app/models/solution.rb

```diff
@@ -79,6 +79,10 @@ def self.for(*args)
 
   delegate :instructions, :introduction, :tests, :source, :source_url, to: :git_exercise
 
+  def external_download_cmd
```

Comment:
I don't know how to name this correctly. We currently have two download commands:

`exercism download uuid=uuid` and `exercism download --track=track --exercise=exercise`.

Reply:
Same as v2. `mentor_download_cmd`